acceptance/tests/facts/windows.rb

```diff
@@ -22,7 +22,10 @@
     os_version  = '2012 R2'
     kernel_version = '6.3'
   elsif agent['platform'] =~ /-10/
-    os_version  = /^10\./
+    os_version  = '10'
```

Comment:
Yes! Great find! That's it and you're totally right! Yes, Windows 10 Ent does count at a consumer operating system ("workstation", at least that's what it's [ProductType](https://msdn.microsoft.com/en-us/library/aa394239(v=vs.85).aspx) returns.

So it looks like if that case were to be updated to catch the right version value, it would actually be just as you say where the full and major versions are identical!